utils.pwsh/Setup-Target.ps1

```diff
@@ -29,13 +46,6 @@ function Setup-BuildParameters {
         . $PSScriptRoot/Logger.ps1
     }
 
-    $NumProcessors = (Get-CimInstance Win32_ComputerSystem).NumberOfLogicalProcessors
-
-    if ( $NumProcessors -gt 1 ) {
-        $env:UseMultiToolTask = $true
-        $env:EnforceProcessCountAcrossBuilds = $true
-    }
-
```

Comment:
Was this moved somewhere, or is it no longer needed?

Reply:
This feedback thread is still open.

Reply:
It's not needed anymore because the same is achieved via the automatic CMake postfix and for non-CMake based projects it's disabled because `make` triggers a single invocation of `cl.exe` for every source file anyway.

Reply:
Thanks for explaining.